Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs

```diff
@@ -404,25 +404,24 @@ public void AcceptToMemoryPool_WithSegWitValidTxns_IsSuccessfull()
             // 50 Coins come from first tx on chain - send bob 42 and change back to miner
             ScriptCoin witnessCoin = new ScriptCoin(context.SrcTxs[0].GetHash(), 0, context.SrcTxs[0].TotalOut, miner.PubKey.ScriptPubKey.WitHash.ScriptPubKey.Hash.ScriptPubKey, miner.PubKey.ScriptPubKey);
             TransactionBuilder txBuilder = new TransactionBuilder();
-            Transaction p2wshTx = txBuilder
+            Transaction p2shOverp2wpkh = txBuilder
```

Comment:
Argh this can get so confusing `miner.PubKey.ScriptPubKey.WitHash.ScriptPubKey.Hash.ScriptPubKey`
It would be a good idea to put an example of how the script looks like for future devs that visit this code.